docker/docker/init/alter_summary_tables.sql

```diff
@@ -0,0 +1,41 @@
+ALTER TABLE event_summary_totals
```

Comment:
remove this file, see ala-install where the identical primary key is created

Reply:
Done